### PR TITLE
Fix #567

### LINKED ISSUE #567: "Projects using Ocean" first paragraph is erroneous
**Describe the bug**
The first paragraph, right under the title, says: "These are live projects that leverage core functionality of Ocean, such as reading or writing on Ocean datatokens or data pools. They don't rely on Ocean Market for an interface, by either having their own GUI or CLI, or using another app."

It says the exact same text 3 paragraphs later in the "Using the Ocean Protocol stack" section.

**To Reproduce**
Go to https://docs.oceanprotocol.com/concepts/projects-using-ocean, you will see it

**Expected behavior**
Different text for first paragraph

### PR DESCRIPTION
**Pull request by trentmc**
Fixes #567.

Changes proposed in this PR:

- Edit text content

### CODE REVIEW

File: content/concepts/projects-using-ocean.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ These are live projects that leverage core functionality of Ocean, such as readi
 
 ## Using OCEAN
 
-These projects use just the Ocean Token, but do more than just exchanging it. This includes staking, and liquidity mining opportunities.
+These projects use just OCEAN token, but do more than just exchanging it. This includes staking, and liquidity mining opportunities.
```
Comment (kremalicious) [inline content/concepts/projects-using-ocean.md:51]: I think we had this standard of using either the ERC-20 name, or the symbol but not sure we stayed that consistent across docs:

- "Ocean Token"
- "OCEAN"

File: content/concepts/projects-using-ocean.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Projects using Ocean Protocol
-description: These are live projects that leverage core functionality of Ocean, such as reading or writing on Ocean datatokens or data pools. They don't rely on Ocean Market for an interface, by either having their own GUI or CLI, or using another app.
+description: These are live projects that use leverage core functionality of Ocean, use Ocean Market, or use OCEAN token with staking and the like.
```
Comment (kremalicious) [inline content/concepts/projects-using-ocean.md:3]: > These are live projects that use leverage...

The "leverage" here seems as it was supposed to be deleted?

File: content/concepts/projects-using-ocean.md (hunk 1 above)
Comment (trentmc) [reply content/concepts/projects-using-ocean.md]: Good eye, I made it all consistently to "OCEAN" here.

File: content/concepts/projects-using-ocean.md (hunk 2 above)
Comment (trentmc) [reply content/concepts/projects-using-ocean.md]: Good eye, fixed

**Review by kremalicious — APPROVED** [review]

**Pull request merged**